engine/execution/provider/engine_test.go

```diff
@@ -28,30 +28,13 @@ import (
 func TestProviderEngine_onChunkDataRequest(t *testing.T) {
 
 	t.Run("non-existent chunk", func(t *testing.T) {
-		ps := mockprotocol.NewState(t)
-		net := mocknetwork.NewNetwork(t)
+		net := new(mocknetwork.Network)
```

Comment:
why change this from `mocknetwork.NewNetwork(t)`? Using the `New*` method automatically asserts expectations which I think is a good practice